packages/remote-feature-flag-controller/src/client-config-api-service/client-config-api-service.test.ts

```diff
@@ -132,43 +191,8 @@ describe('ClientConfigApiService', () => {
       // Check that fetch was retried the correct number of times
       expect(mockFetch).toHaveBeenCalledTimes(maxRetries + 1); // Initial + retries
     });
-  });
-
-  describe('circuit breaker', () => {
-    it('opens the circuit breaker after consecutive failures', async () => {
```

Comment:
Similar to tests in `codefi-v2.test.ts`, we don't need these since they're already covered by the `createServicePolicy` tests.